__tests__/app.test.js

```diff
@@ -128,3 +127,59 @@ describe("GET /api/articles", () => {
     });
 });
 })
+describe("GET /api/articles/:article_id/comments ", () => {
+  test("200: Responds with an array of comments for a given article_id", () => {
+    return request(app)
+    .get("/api/articles/9/comments")
+    .expect(200)
+    .then(({ body }) => {
+      const { comments } = body;
+      expect(comments).toHaveLength(2);
+      comments.forEach((comment) => {
+        expect(comment).toMatchObject({
+          comment_id: expect.any(Number),
+          votes: expect.any(Number),
+          created_at: expect.any(String),
+          author: expect.any(String),
+          body: expect.any(String),
+          article_id: 9
```

Comment:
Nice! Well done for making sure the test is still flexible but that we definitely have received back the comments related to the correct article regardless of what the comments info is!